--- a/python/sglang/srt/model_executor/model_runner.py
+++ b/python/sglang/srt/model_executor/model_runner.py
@@ -1525,7 +1525,7 @@
     def init_lora_manager(self):
         self.lora_manager = LoRAManager(
             base_model=self.model,
-            base_hf_config=self.model_config.hf_config,
+            base_hf_config=self.model_config.hf_text_config,
             max_loras_per_batch=self.server_args.max_loras_per_batch,
             load_config=self.load_config,
             dtype=self.dtype,
